atc/exec/get_step.go

```diff
@@ -121,7 +118,8 @@ func (step *GetStep) Run(ctx context.Context, state RunState) error {
 
 	step.delegate.Initializing(logger)
 
-	variables := creds.NewVariables(step.secrets, step.metadata.TeamName, step.metadata.PipelineName)
+	//variables := creds.NewVariables(step.secrets, step.metadata.TeamName, step.metadata.PipelineName)
+	variables := step.delegate.GetCredVarsTracker()
```

Comment:
Stylistically we prefer to not have `Get`-prefixed methods - can we just rename this to `Variables()` and have it return `creds.Variables` interface since that's the only interface the steps care about?